<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageCoursesPage</class>
 <widget class="QDialog" name="ManageCoursesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1606</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0D1B2A;
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 32pt &quot;Segoe UI&quot;;



</string>
   </property>
   <property name="text">
    <string>Manage Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="addNewCourseLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 20pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>+ Add New Course</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="courseNameInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>1171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1E293B;
    color: #E2E8F0;
    border: 1px solid #334155;
    border-radius: 4px;
    padding: 8px;
    font-size: 12px;
}

QLineEdit:focus {
    border: 2px solid #3B82F6;   
    background-color: #273449;   
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dayCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>1171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #1E293B;
    color: #E2E8F0;
    border: 1px solid #334155;
    border-radius: 4px;
    padding: 8px;
    font-size: 12px;
}

/* Â±ïÂºÄÈ°πÁõÆ */
QComboBox QAbstractItemView {
    background-color: #1E293B;
    color: #E2E8F0;
    selection-background-color: #3B82F6;
    selection-color: white;
}

QComboBox:focus {
    border: 2px solid #3B82F6;
    background-color: #273449;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="startTimeLabel_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Start Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dayLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Day:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>End Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classroomInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>1171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1E293B;
    color: #E2E8F0;
    border: 1px solid #334155;
    border-radius: 4px;
    padding: 8px;
    font-size: 12px;
}

QLineEdit:focus {
    border: 2px solid #3B82F6;   
    background-color: #273449;   
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="classroomLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Classroom:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCourseBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>1171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 5px;
    padding: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #27ae60;
}

QPushButton:pressed {
    background-color: #1e874b;
}</string>
   </property>
   <property name="text">
    <string>+ Add Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="courseCountLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 32pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>View &amp; Manage Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #9b59b6;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 5px;
    padding: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #8e44ad;
}

QPushButton:pressed {
    background-color: #7d3c98;
}</string>
   </property>
   <property name="text">
    <string>‚è±Ô∏è Generate Timetable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewTimetableBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>670</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ff8c42;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 5px;
    padding: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #e67e22;
}

QPushButton:pressed {
    background-color: #d35400;
}</string>
   </property>
   <property name="text">
    <string>üìÖ View Generated Timetables</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Course NameÔºö</string>
   </property>
  </widget>
  <widget class="QComboBox" name="startTimeLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #1E293B;
    color: #E2E8F0;
    border: 1px solid #334155;
    border-radius: 4px;
    padding: 8px;
    font-size: 12px;
}

/* Â±ïÂºÄÈ°πÁõÆ */
QComboBox QAbstractItemView {
    background-color: #1E293B;
    color: #E2E8F0;
    selection-background-color: #3B82F6;
    selection-color: white;
}

QComboBox:focus {
    border: 2px solid #3B82F6;
    background-color: #273449;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="endTimeInput">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>250</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #1E293B;
    color: #E2E8F0;
    border: 1px solid #334155;
    border-radius: 4px;
    padding: 8px;
    font-size: 12px;
}

/* Â±ïÂºÄÈ°πÁõÆ */
QComboBox QAbstractItemView {
    background-color: #1E293B;
    color: #E2E8F0;
    selection-background-color: #3B82F6;
    selection-color: white;
}

QComboBox:focus {
    border: 2px solid #3B82F6;
    background-color: #273449;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logupbutton">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 350 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Logup</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="coursetable">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>1171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
background-color: #f0f0f0;
alternate-background-color: #e0e0e0;
gridline-color: #d0d0d0;
font-family: Arial;
font-size: 12px;
}

QTableWidget::item {
padding: 5px;
border: 1px solid #d0d0d0;
}

QTableWidget::item:selected {
background-color: #a0a0a0;
}

QHeaderView::section {
background-color: #4CAF50;
color: white;
padding: 5px;
border: 1px solid #4CAF50;
font-weight: bold;
}</string>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Select</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Course Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Day</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Classroom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Actions</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
